scripted-tests/run/java-net-socket/build.sbt

```diff
@@ -0,0 +1,30 @@
+import java.net.{ServerSocket}
+import java.io.{PrintWriter, BufferedReader, InputStreamReader}
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+enablePlugins(ScalaNativePlugin)
+
+scalaVersion := "2.11.11"
+
+lazy val launchEchoServer =
+  taskKey[Unit]("Setting up echo server on port 5832")
+
+launchEchoServer := {
+  val echoServer = new ServerSocket(5832)
```

Comment:
It would probably be better to let the JVM select a port, write it to a file and read the port from the file on the native side (otherwise I'm afraid we could have spurious failures).